lms/djangoapps/certificates/models.py

```diff
@@ -842,26 +842,38 @@ def course_key(self):
 class CertificateGenerationCourseSetting(TimeStampedModel):
     """Enable or disable certificate generation for a particular course.
 
-    This controls whether students are allowed to "self-generate"
+    DEPRECATED: 'enabled' controls whether students are allowed to "self-generate"
     certificates for a course.  It does NOT prevent us from
     batch-generating certificates for a course using management
     commands.
 
+    'self_generation_enabled' should be used in place of 'enabled'
+
     In general, we should only enable self-generated certificates
     for a course once we successfully generate example certificates
     for the course.  This is enforced in the UI layer, but
     not in the data layer.
 
+    'language_specific_templates_enabled' controls whether the course can
+    use templates for specific languages. language should be specified in
+    the CertificateTemplate
+
     """
     course_key = CourseKeyField(max_length=255, db_index=True)
-    enabled = models.BooleanField(default=False)
+    enabled = models.BooleanField(default=False)  # deprecated
+    # note: accessor methods below for self_generation_enabled
```

Comment:
Create a ticket, and add its identifier to all of these comments. Also, use `# TODO` instead of `# note` so that IDEs will pick them up.